modules/api/php/endpoints/candidate/visit/visit.class.inc

```diff
@@ -12,6 +12,7 @@
  */
 namespace LORIS\api\Endpoints\Candidate\Visit;
 
+use LORIS\api\Endpoints\Candidate\Visit\Electrophysiology\Electrophysiology;
```

Comment:
This do not seem to be used

Reply:
ooops... Indeed. Leftover from tries at the beginning ;)